Source/Core/DolphinQt/GBAWidget.cpp

```diff
@@ -166,6 +166,23 @@ void GBAWidget::UnloadROM()
   RestartCore(m_core);
 }
 
+void GBAWidget::QueueEReaderCard()
+{
+  QString card_path = QDir::toNativeSeparators(QFileDialog::getOpenFileName(
```

Comment:
This should call ``QFileDialog::getOpenFileNames`` instead to add multiple files at once for convenience

Reply:
Yeah, as e-Reader cards are in pairs, loading two at once would probably be nice.